_fixtures/goroutinestackprog.go

```diff
@@ -9,7 +9,10 @@ func agoroutine(started chan<- struct{}, done chan<- struct{}, i int) {
 	done <- struct{}{}
 }
 
+var dummy int
```

Comment:
This change broke `TestStacktraceGoroutine`, that test needs to be updated to reflect the new line numbering.